lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2286,6 +2286,10 @@ definitions:
           - 'Baicells ID TDD/FDD'
           - 'NuRAN Cavium OC-LTE'
         x-nullable: false
+      mme_ip:
+        type: string
+        format: ipv4
+        example: '10.0.2.1'
```

Comment:
maybe you can set a default value to empty string
```
default: ""
```

Reply:
Maybe I can set it "" , but I hope that in the swagger api ,the doc can tell the user what does the value mean?

Reply:
> Requested some changes, may I ask what is the long term plan to add this config value? With this change we will add NMS support to stream this value and display it, but it doesn’t change any of the actual device config, is this planned for configuring managed radios?
> 
> if you could add some context on the Summary section that would be good to follow up from there, thanks!

OK, I will add the Summary section.